packages/react-server-cli/src/commands/compile.js

```diff
@@ -0,0 +1,23 @@
+import compileClient from "../compileClient"
+import handleCompilationErrors from "../handleCompilationErrors";
+import setupLogging from "../setupLogging";
+import logProductionWarnings from "../logProductionWarnings";
+
+const logger = require("react-server").logging.getLogger(__LOGGER__);
```

Comment:
I thought we were trying to set the pattern of `import {logging} from 'react-server'; const logger = loggin.getLogger(__LOGGER__);`?  Or is that a non-goal?